### ISSUE TITLE
Not able to fetch pre-processed dataset

### ISSUE THREAD

**Issue by hemangjethava** [osin-vladimir/object-detection#1]
Can you help me out to download training dataset of your git, Pascal's official website is not providing separate train and validation set folders. There may be something i may have missed for preprocessing VOC2007 to convert it into train and validation set folders.

**Comment by hemangjethava** [comment]
ignore the question! I am new to Keras so didn't get any idea. hence, closing the issue!!
